cli/web3cli/test_inventory.py

```diff
@@ -0,0 +1,2165 @@
+            to_block=tx_receipt_1.block_number,
+        )
+        self.assertEqual(len(item_equipped_events), 2)
+
+        self.assertEqual(
+            item_equipped_events[0]["args"]["subjectTokenId"], subject_token_id
+        )
+        self.assertEqual(item_equipped_events[0]["args"]["slot"], slot)
+        self.assertEqual(
+            item_equipped_events[0]["args"]["itemType"],
+            20,
+        )
+        self.assertEqual(
+            item_equipped_events[0]["args"]["itemAddress"],
+            self.payment_token.address,
+        )
+        self.assertEqual(
+            item_equipped_events[0]["args"]["itemTokenId"],
+            0,
+        )
+        self.assertEqual(
+            item_equipped_events[0]["args"]["amount"],
+            2,
+        )
+        self.assertEqual(
+            item_equipped_events[0]["args"]["equippedBy"],
+            self.player.address,
+        )
+
+        self.assertEqual(
+            item_equipped_events[1]["args"]["subjectTokenId"], subject_token_id
+        )
+        self.assertEqual(item_equipped_events[1]["args"]["slot"], slot)
+        self.assertEqual(
+            item_equipped_events[1]["args"]["itemType"],
+            1155,
+        )
+        self.assertEqual(
+            item_equipped_events[1]["args"]["itemAddress"],
+            self.terminus.address,
+        )
+        self.assertEqual(
+            item_equipped_events[1]["args"]["itemTokenId"],
+            item_pool_id,
+        )
+        self.assertEqual(
+            item_equipped_events[1]["args"]["amount"],
+            9,
+        )
+        self.assertEqual(
+            item_equipped_events[1]["args"]["equippedBy"],
+            self.player.address,
+        )
+
+        item_unequipped_events = _fetch_events_chunk(
+            web3_client,
+            inventory_events.ITEM_UNEQUIPPED_ABI,
+            from_block=tx_receipt_1.block_number,
+            to_block=tx_receipt_1.block_number,
+        )
+        self.assertEqual(len(item_unequipped_events), 1)
+        self.assertEqual(
+            item_unequipped_events[0]["args"]["subjectTokenId"], subject_token_id
+        )
+        self.assertEqual(item_unequipped_events[0]["args"]["slot"], slot)
+        self.assertEqual(
+            item_unequipped_events[0]["args"]["itemType"],
+            20,
+        )
+        self.assertEqual(
+            item_unequipped_events[0]["args"]["itemAddress"],
+            self.payment_token.address,
+        )
+        self.assertEqual(
+            item_unequipped_events[0]["args"]["itemTokenId"],
+            0,
+        )
+        self.assertEqual(
+            item_unequipped_events[0]["args"]["amount"],
+            2,
+        )
+        self.assertEqual(
+            item_unequipped_events[0]["args"]["unequippedBy"],
+            self.player.address,
+        )
+
+    def test_player_cannot_unequip_erc20_tokens_from_persistent_slot_but_can_increase_amount(self):
+        """
```

Comment:
I think we should change the test name to indicate the test is broken or just comment out the whole test for now. I read through equip / _unequip methods a couple times before I noticed the comment that noted this test was unfinished as the functionality was missing.

Reply:
I renamed the test to `test_player_cannot_unequip_erc20_tokens_from_persistent_slot`.

Reply:
It's still a useful test as it tests persistence of slots.